hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -42,6 +42,7 @@
 
   public static final String STORAGE_ID = "storageID";
   public static final String DATANODE_UUID = "datanodeUuid";
+  public static final String DATANODE_STORAGE_DIR = "datanodeStorageConfig";
```

Comment:
There is also a "datanode.id" file being created in the Datanode that looks like a metadata yaml file. Can we look into whether we can add our layout version into that file? Reference -> _org.apache.hadoop.hdds.utils.HddsServerUtil#getDatanodeIdFilePath_

Reply:
yup, storage config is used just for the upgrade and nothing else. We will continue to use datanode.id file as it is.